### PR TITLE
Added slack link

### LINKED ISSUE #99: Add link to Slack
Seeing that we have opted for Slack, I'd suggest that we make the Slack signup easier to find by adding a shortcut link.

### PR DESCRIPTION
**Pull request by jesperlindstrom**
Closes #99 

skarmavbild 2016-11-08 kl 20 00 14

### CODE REVIEW

**Comment by lindskogen** [comment]
You can use https://slack.chalmers.it/signup

**Comment by jesperlindstrom** [comment]
Fixed

**Comment by lindskogen** [comment]
Also, should you really use signup link for everyone? Many already have accounts?

**Comment by lindskogen** [comment]
Nvm, both seem to work :)

**Comment by jesperlindstrom** [comment]
Good question. I considered the login page first, but the signup link wasn't very visible on that page so I assumed people who want to sign up might get confused. Perhaps an alternative would be to link to a separate info page where we explain what Slack is (similar to irc.chalmers.it) and also link to both signup and login... Might be overkill though, not sure.

File: app/views/layouts/home.html.erb [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
         <li><a data-tooltip title="TimeEdit" href="https://se.timeedit.net/web/chalmers/db1/public/ri1Q7.html" target="_blank" class="has-tip tip-top"><%= fa_icon 'table', text: t('.schema') %></a></li>
         <li><a data-tooltip title="<%= t '.book_hubben_desc' %>" href="https://bookit.chalmers.it/bookings/new" target="_blank" class="has-tip tip-top"><%= fa_icon 'home', text: t('.booking') %></a></li>
         <li><a data-tooltip title="<%= t '.book_chalmers_desc' %>" href="https://se.timeedit.net/web/chalmers/db1/b1/" target="_blank" class="has-tip tip-top"><%= fa_icon 'users', text: t('.group_room') %></a></li>
         <li><a data-tooltip title="<%= t '.print_desc' %>" href="<%= new_print_path %>" class="has-tip tip-top"><%= fa_icon 'print', text: t('.print') %></a></li>
         <li><a data-tooltip title="<%= t '.charge_card_desc' %>" href="http://kårkort.se" target="_blank" class="has-tip tip-top"><%= fa_icon 'credit-card', text: t('.charge_card') %></a></li>
+        <li><a data-tooltip title="<%= t '.slack_desc' %>" href="https://slack.chalmers.it/signup" target="_blank" class="has-tip tip-top"><%= fa_icon 'slack', text: t('.slack') %></a></li>
```
Comment (lindskogen) [inline app/views/layouts/home.html.erb]: Did you check how it looks on mobile? That layout broke easily by my experience...

Comment (jesperlindstrom) [inline app/views/layouts/home.html.erb]: Just checked. Works alright with Swedish, but English has some longer text here. Will need to fix

**Comment by jesperlindstrom** [comment]
skarmavbild 2016-11-11 kl 14 06 06
Changed to a more robust solution where they're split into a 3x2 grid, hard to fit them all in one row on a small screen.

**Pull request merged**